modules/hydrodyn/src/Morison.f90

```diff
@@ -2126,7 +2117,7 @@ SUBROUTINE Morison_Init( InitInp, u, p, x, xd, z, OtherState, y, m, Interval, In
       tMG       = -999.0
       An_drag   = 0.0
       
-      IF ( (InitInp%InpJoints(i)%Position(3)-p%MSL2SWL) >= -p%WtrDpth ) THEN
+      IF ( (InitInp%InpJoints(i)%Position(3)-p%WaveField%MSL2SWL) >= -InitInp%WaveField%EffWtrDpth ) THEN !bjj: ask Lu if this is correct. I wonder if this check is supposed to be against WtrDpth
```

Comment:
This should be ok. Based on my notes, `InitInp%InpJoints(i)%Position(3)` is measured from the MSL. Alternatively, we can simplify the check to `InitInp%InpJoints(i)%Position(3) >= -InitInp%WaveField%WtrDpth`.

Reply:
Addressed in 67c77c: I updated the check to be `InitInp%InpJoints(i)%Position(3) >= -InitInp%WaveField%WtrDpth`. I think that's easier to read.